src/Compilers/CSharp/Portable/Binder/Binder_Attributes.cs

```diff
@@ -819,6 +821,14 @@ private TypedConstant GetDefaultValueArgument(ParameterSymbol parameter, Attribu
                 kind = TypedConstantKind.Primitive;
                 defaultValue = ((ContextualAttributeBinder)this).AttributedMember.GetMemberCallerName();
             }
+            else if (!IsEarlyAttributeBinder && syntax.ArgumentList is not null &&
+                getCallerArgumentArgumentIndex(parameter, argumentsToParams) is int argumentIndex && argumentIndex > -1 && argumentIndex < syntax.ArgumentList.Arguments.Count)
+            {
+                CheckFeatureAvailability(syntax.ArgumentList, MessageID.IDS_FeatureCallerArgumentExpression, diagnostics);
```

Comment:
>CheckFeatureAvailability [](http://example.com/codeflow?start=16&length=24)

It doesn't look like we are testing effect of this call. In fact, I also don't see an existing test covering the same scenario for a regular invocation (looked only in the modified test file).